app/controllers/concerns/foreman/controller/parameters/user_notification.rb

```diff
@@ -0,0 +1,15 @@
+module Foreman::Controller::Parameters::UserNotification
+  extend ActiveSupport::Concern
```

Comment:
Just tidiness, *all* controllers have their parameters extended through concerns in this directory. Additionally the plugin interface needs this to exist to add more parameters to the list